rskj-core/src/main/java/co/rsk/trie/TrieImpl.java

```diff
@@ -237,7 +236,7 @@ private static TrieImpl fromMessage(byte[] message, int position, int msglength,
         } catch (IOException ex) {
             logger.error(ERROR_CREATING_TRIE, ex);
             panicProcessor.panic(PANIC_TOPIC, ERROR_CREATING_TRIE +": " + ex.getMessage());
-            throw new TrieSerializationException(ERROR_CREATING_TRIE, ex);
+            throw new RuntimeException(ERROR_CREATING_TRIE, ex);
```

Comment:
I agree with Sonar on this. I think it's a better practice to try to be somehow more specific when throwing RuntimeExceptions. In this case, another `IllegalStateException` may apply